wiki/Installation/macOS/en.md

```diff
@@ -1,20 +1,43 @@
 # Installation on macOS
 
-This page will roughly tell you how to get osu! working on your macOS device.
+This page will roughly guide you on getting osu! working on your macOS device.
 
 ## Minimum system requirements
 
 - A decent computer running macOS.
 
 ## Installing osu!
 
-The macOS version of osu! is experimental. The game may not function properly or even boot at all. This [tutorial video](https://www.youtube.com/watch?v=3ay221bMHNo) will help you understand the process below.
+The macOS version of osu! is subject to minor display bugs and less performant gameplay. The game may not function perfectly at first and takes some fine-tuning to get used to. To install it:
 
-1. Go to the unofficial [download page](https://osu.ppy.sh/community/forums/topics/682197), download `osu!.app.zip` and unzip it.
-2. You will see the `osu!.app` folder, which will look like an application. Right click it, and select `Open`.
-3. The installer will automatically run. The installation location cannot be changed and will be installed directly to `osu!.app`. Moving `osu!.app` will move all of the files that come with it. You can also right click it and select `Show Package Contents` to view the files within the folder.
-4. When the installer is completed, osu! will start up right after. osu!direct will be used to download some starter beatmaps for you.
-   - If osu! is blocked from opening due to having an unidentified developer, open `System Preferences > Security and Privacy` and click `Open Anyway`.
-5. osu! will prompt you to sign in or register.
+1. Go to the unofficial Wineskin [download page](https://osu.ppy.sh/community/forums/topics/1106057), download the latest Wineskin and unzip it.
+2. Download the unofficial [osu!macOS Agent](https://osu.ppy.sh/community/forums/topics/1036678) tool and use it to repair the `osu!.app` file. Alternatively:
+   1. Move `osu!.app` to the desktop.
+   2. Open Terminal, copy this command: `xattr -c 'Desktop/osu!.app'`, and press enter.
```

Comment:
The `Return` word looks bigger on my keyboard so that seems right. I think it'd be better to make it something like:

> and press the `Return` key.

so it doesn't get confused with some UI element named "Return". That being said, I don't know how many users are familiar with the key being called `Return` rather than `Enter`.